pymchelper/run.py

```diff
@@ -57,6 +58,11 @@ def main(args=sys.argv[1:]):
 
     set_logger_level(parsed_args)
 
+    # check if output directory exists
+    output_dir = os.path.dirname(parsed_args.output)
+    if not(os.path.exists(output_dir)):
+        raise IOError("Directory {}/ does not exist.".format(output_dir))
```

Comment:
Output might also be a filename, for example:
`convertmc NBDoseDD0001.bdo nice_plot.png --converter image `

then `parsed_args.output` is equal to nice_plot.png and it is expected that this file might not exists.